components/Api/JsonLink/index.stories.ts

```diff
@@ -0,0 +1,14 @@
+import JsonLink from './index';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof JsonLink>;
+type Meta = MetaObj<typeof JsonLink>;
+
+export const Default: Story = {
+  args: {
+    version: 'v18',
+    fileName: 'documentation',
+  },
+};
+
+export default { component: JsonLink } as Meta;
```

Comment:
I see that you've added a story for the component. Thanks for doing that. We're also saving snapshots of our stories so you can run `npm run test:storybook:snapshot` to generate that, and you can commit that too.

Reply:
Thank you - totally missed that, added now.